controllers/topics/index.js

```diff
@@ -31,6 +31,7 @@ const {getAnonymUser} = require('../../utils/getAnonymUser');
 const UsersFunction = require('../../databases/functions/users');
 const {insertSingleTopic} = require('../../utils/post');
 const {followTopicServiceQueue} = require('../../services/redis');
+const broadcastMessage = require('./broadcastMessage');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `broadcastMessage` module is imported but it's not clear if error handling is properly implemented within the module. Ensure that the function handles possible errors and doesn't propagate them up the call stack, causing potential unhandled promise rejections.
```diff
- const broadcastMessage = require('./broadcastMessage');
+ const broadcastMessage = require('./broadcastMessage'); // ensure this module has proper error handling
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->